src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/PtrToStringTests.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Runtime.InteropServices.Tests
+{
+    public class PtrToStringTests
+    {
+        [Fact]
+        public void PtrToStringAnsi()
+        {
+            AssertExtensions.Throws<ArgumentNullException>("ptr", () => Marshal.PtrToStringAnsi(IntPtr.Zero, 123));
+            Assert.Throws<ArgumentException>(() => Marshal.PtrToStringAnsi(new IntPtr(123), -77));
+        }
+
+        [Fact]
+        public void PtrToStringUni()
+        {
+            AssertExtensions.Throws<ArgumentNullException>("ptr", () => Marshal.PtrToStringUni(IntPtr.Zero, 123));
+            Assert.Throws<ArgumentException>(() => Marshal.PtrToStringUni(new IntPtr(123), -77));
+        }
+
+        [Fact]
+        public void PtrToStringAuto()
+        {
+            AssertExtensions.Throws<ArgumentNullException>("ptr", () => Marshal.PtrToStringAuto(IntPtr.Zero, 123));
+            Assert.Throws<ArgumentException>(() => Marshal.PtrToStringAuto(new IntPtr(123), -77));
+        }
+
+        [Fact]
+        public void PtrToStringBSTR()
+        {
+            AssertExtensions.Throws<ArgumentNullException>("ptr", () => Marshal.PtrToStringBSTR(IntPtr.Zero));
+        }
+
+        [Fact]
+        public void PtrToStringUTF8()
+        {
+            Marshal.PtrToStringUTF8(IntPtr.Zero, 123);
```

Comment:
Seems like this is missing an assert, validating something about the returned string?

Reply:
Fixed with `Assert.Null(Marshal.PtrToStringUTF8(IntPtr.Zero, 123));` as this will return null.